### PR TITLE
Adding CLUE barometer project

### PR DESCRIPTION
**Pull request by BlitzCityDIY**
Adding the CLUE barometer project. Uses a servo to display the barometric pressure. Screen displays temperature and pressure readings. A and B buttons can switch between imperial and US units.

### CODE REVIEW

**Review by TrevKnows — CHANGES_REQUESTED** [review]
Would it be possible to change the project title to "CLUE Barometer"?

**Comment by BlitzCityDIY** [comment]
@TrevKnows totally! should i rename the others that i've added too to drop the pyleap?

File: pyleapProjects.json [hunk 1]
```diff
@@ -166,6 +166,17 @@
           "clue_nrf52840_express"
       ]
     },
+	
+	{
+      "projectName": "PyLeap CLUE Barometer",
+      "projectImage": "https://cdn-learn.adafruit.com/assets/assets/000/115/821/medium640/projects_edited_P1390003.jpg?1665513267",
+      "description": "In this project, you'll turn your CLUE a barometric pressure and temperature monitor. A servo will display the pressure readings.",
```
Comment (caternuson) [inline pyleapProjects.json]: ...turn your CLUE **into** a barometric...

**Review by caternuson — CHANGES_REQUESTED** [review]

**Review by caternuson — APPROVED** [review]

File: pyleapProjects.json (hunk 1 above)
Comment (TrevKnows) [reply pyleapProjects.json]: @BlitzCityDIY yes, when you can please rename the other project titles you've added

**Pull request merged**